src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginsPage.cs

```diff
@@ -0,0 +1,52 @@
+using JetBrains.Annotations;
+using Microsoft.Extensions.DependencyInjection;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.Serialization.Attributes;
+using NexusMods.App.UI.Resources;
+using NexusMods.App.UI.WorkspaceSystem;
+
+namespace NexusMods.App.UI.Pages.ModLibrary;
+
```

Comment:
The namespaces in this entire folder are off. Either disable namespace providing or update the namespace.

Reply:
The FileOrigins subfolder is marked as not providing for namespace. What is the issue?

Reply:
It's not marked

![2024-04-29_14-00-11](https://github.com/Nexus-Mods/NexusMods.App/assets/16577545/7f775a51-5641-4142-adce-ae775812df3e)

Reply:
![image](https://github.com/Nexus-Mods/NexusMods.App/assets/26797547/b744846e-51b2-4315-90a1-204fbbfcfbd6)

Reply:
I removed and added it again and this time the DotSettings file updated properly